src/ddmd/dclass.d

```diff
@@ -242,6 +240,10 @@ extern (C++) class ClassDeclaration : AggregateDeclaration
     /// to prevent recursive attempts
     private bool inuse;
 
+    /// true if this class has an identifier, but was originally declared anonymous
+    /// used in support of https://issues.dlang.org/show_bug.cgi?id=17371
+    private bool isActuallyAnonymous;
```

Comment:
the D convention is, as far as I know, to use the same name as the getter but prefix it with an underscore.

Reply:
I haven't seen that.  Show me where the precedent is established.

Reply:
I don't know where/if it's written down somewhere. But I think the name is a bit strange anyway, having `Actually` as part of the name doesn't add anything.

Reply:
The "Actually" documents the fact that we are lying, because the class has a `__anonclass` identifier.  And remember I had to overload a method called `isAnonymous`, so that name is off limits.

Reply:
> And remember I had to overload a method called isAnonymous, so that name is off limits.

That's why it's common to name the field either `_isAnonymous` or `isAnonymous_`. I prefer the latter but I think the former is more common in Phobos.

Reply:
>  That's why it's common to name the field either _isAnonymous or isAnonymous_

I would prefer something like that as well, but that doesn't seem to be the convention in DMD.

Reply:
No, but in the rest of the D community, at least in the core projects, the ones in the `dlang` GitHub organization. Keep in mind that DMD is quite an old code base written in a very old style, C with classes. We're trying to improve that and in my opinion any new code should follow the standard D conventions.